libs/SmartStore/src/com/salesforce/androidsdk/smartstore/store/DBOpenHelper.java

```diff
@@ -567,7 +567,7 @@ public JSONObject loadSoupBlob(String soupTableName, long soupEntryId, String pa
 		try {
 			result = new JSONObject(loadSoupBlobAsString(soupTableName, soupEntryId, passcode));
 
-		} catch (JSONException ex) {
```

Comment:
Tests are failing because the exception type is NullPointException here if the soupBlob is deleted (so instead of a mal-format json, it's a null)